tests/workers/docker.go

```diff
@@ -14,6 +14,9 @@ func InitDockerWorker() {
 	integration.Register(&dockerWorker{
 		id: "docker",
 	})
+	integration.Register(&dockerWorker{
+		id: "docker+containerd",
```

Comment:
We should pass the value for `ContainerdSnapshotter` in here, instead of deriving it from the name.

See [moby/buildkit@`849d014` (#3866)](https://github.com/moby/buildkit/pull/3866/commits/849d01449016fb6613a0b3799ebbf64f0ac4772f) for similarly in buildkit.